### ISSUE TITLE
Making temperature adjustment less noticeable (more frequent adjustments)

### ISSUE THREAD

**Issue by diamondavocado** [Tyrrrz/LightBulb#54]
After switching to Lightbulb from Flux, I'm enjoying it a lot, but I really notice the small temperature adjustments during the transitional period. 

I'm going from max daytime temp. to minimum night time temp. over the maximum allowed period (3 hours) and every few minutes I can see the screen temperature step down which can be annoying.

Is there a way to increase the granularity of the temperature adjustment so that it adjusts more smoothly during the transitional period?

**Comment by Tyrrrz** [comment]
There's a config file adjustment you can make for that right now.
https://github.com/Tyrrrz/LightBulb/wiki/Config-file
It's the `TemperatureUpdateInterval`, default is 1 minute.

**Comment by diamondavocado** [comment]
Awesome, thank you. I've set this to 1 second. 

Would this have any noticeable impact on performance? If not, may I suggest making the default as low as possible so the transition is as smooth as possible?

**Comment by Tyrrrz** [comment]
yes it does make an impact

**Comment by diamondavocado** [comment]
This isn't actually working as expected. I've got my interval set to 1 second as discussed, but the temperature is still not changing smoothly. It still only changes about every minute, and I just saw it jump from 2750k to 2800k when I would expect it to change much more gradually than that.

Is there a minimum change level of 50 units or something?

My config file:

`{
  "IsGammaPollingEnabled": true,
  "IsTemperatureSmoothingEnabled": true,
  "IsFullscreenBlocking": true,
  "IsCheckForUpdatedEnabled": true,
  "IsInternetSyncEnabled": false,
  "IsGeoInfoLocked": false,
  "TemperatureEpsilon": 50,
  "DefaultMonitorTemperature": 6600,
  "MinTemperature": 2500,
  "MaxTemperature": 6600,
  "MaximumTemperatureSmoothingDuration": "00:00:03",
  "TemperatureTransitionDuration": "05:00:00",
  "TemperatureUpdateInterval": "00:00:01",
  "GammaPollingInterval": "00:00:05",
  "CheckForUpdatesInterval": "06:00:00",
  "InternetSyncInterval": "06:00:00",
  "SunriseTime": "11:00:00",
  "SunsetTime": "20:00:00",
  "GeoInfo": null,
  "ToggleHotkey": null,
  "TogglePollingHotkey": null,
  "Proxy": null
}`
